### PR TITLE
Fix gene agg search and display

### PR DESCRIPTION
**Pull request by hanars**

### CODE REVIEW

File: seqr/utils/elasticsearch/es_gene_agg_search.py [hunk 1]
```diff
@@ -79,8 +79,15 @@ def _parse_response(self, response):
 
                 for key in HAS_ALT_FIELD_KEYS:
                     for sample_agg in gene_agg[key]['buckets']:
-                        family_guid = families_by_sample[sample_agg['key']]
-                        gene_counts[gene_id]['families'][family_guid] += sample_agg['doc_count']
+
```
Comment (ShifaSZ) [inline seqr/utils/elasticsearch/es_gene_agg_search.py:82]: Unnecessary blank line.

File: seqr/utils/elasticsearch/es_utils_tests.py [hunk 2]
```diff
@@ -2479,6 +2482,42 @@ def test_multi_project_all_samples_all_inheritance_get_es_variant_gene_counts(se
...
         self.assertCachedResults(results_model, {'gene_aggs': gene_counts, 'total_results': 5})
 
+    @urllib3_responses.activate
+    def test_all_samples_any_affected_get_es_variant_gene_counts(self):
+        setup_responses()
+        search_model = VariantSearch.objects.create(search={
+            'annotations': {'frameshift': ['frameshift_variant']}, 'inheritance': {'mode': 'any_affected'},
+        })
+        results_model = VariantSearchResults.objects.create(variant_search=search_model)
+        results_model.families.set(Family.objects.filter(project__guid='R0001_1kg'))
```
Comment (ShifaSZ) [inline seqr/utils/elasticsearch/es_utils_tests.py:2492]: Could the data for VariantSearchResults be put into the fixture data?

**Review by ShifaSZ — CHANGES_REQUESTED** [review]
There are two comments.

File: seqr/utils/elasticsearch/es_utils_tests.py (hunk 2 above)
Comment (hanars) [reply seqr/utils/elasticsearch/es_utils_tests.py:2492]: It isn't in the other 29 tests in the file, so that would be an anti-pattern

**Comment by hanars** [comment]
passing tests: https://travis-ci.org/github/broadinstitute/seqr/builds/757140847

**Review by ShifaSZ — APPROVED** [review]

**Pull request merged**